internal/component/remote/vault/vault_test.go

```diff
@@ -142,6 +143,79 @@ func Test_PollSecrets(t *testing.T) {
 	}
 }
 
+func Test_PollSecretsWithKey(t *testing.T) {
+	var (
+		ctx = componenttest.TestContext(t)
+		l   = util.TestLogger(t)
+	)
+
+	cli := getTestVaultServer(t)
+
+	// Store a secret in value to use from the component.
+	_, err := cli.KVv2("secret").Put(ctx, "test", map[string]any{
+		"key": "value",
+	})
+	require.NoError(t, err)
+
+	cfg := fmt.Sprintf(`
+		server = "%s"
+		path   = ""
```

Comment:
Should this have a path to more accurately test the issue? Should it be `secret` and key be `test`?